transforms/language/pdf2parquet/dpk_pdf2parquet/ray/s3_ray.py

```diff
@@ -16,7 +16,7 @@
 
 from data_processing.utils import ParamsUtils
 from data_processing_ray.runtime.ray import RayTransformLauncher
```

Comment:
`data_processing_ray` is available only if dpk is installed with the `[ray]` extra, right?

we could wrap these imports in `try: ... except ImportError: raise ImportError("Please install data_prep_kit[ray]")` or similar.

Reply:
Done! Please resolve conversation and approve. Thanks